Solutions/MimecastTIRegional/SolutionMetadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "publisherId": "mimecast",
+    "offerId": "azure-sentinel-solution-mimecasttiregional",
+    "firstPublishDate": "2022-02-24",

```

Comment:
Is the first published data ok??

Reply:
I will update that actually the was raised previously on the mentioned date but it was closed without merging.